source/github-helpers/dom-formatters.ts

```diff
@@ -36,11 +35,13 @@ export function linkifyIssues(
 	// Enable native issue title fetch
 	for (const link of linkified.children as HTMLCollectionOf<HTMLAnchorElement>) {
 		const issue = link.href.split('/').pop();
-		link.setAttribute('class', 'issue-link js-issue-link tooltipped tooltipped-ne');
-		link.dataset.errorText = 'Failed to load issue title';
-		link.dataset.permissionText = 'Issue title is private';
+		link.setAttribute('class', 'issue-link js-issue-link');
+		link.dataset.errorText = 'Failed to load title';
+		link.dataset.permissionText = 'Title is private';
 		link.dataset.url = link.href;
 		link.dataset.id = `rgh-issue-${issue!}`;
+		link.dataset.hovercardType = 'issue';
+		link.dataset.hovercardUrl = `${link.pathname}/hovercard`;
```

Comment:
I've said this before but: GitHub's routing is _super-duper_ smart so it works even if the path has a trailing slash.